homeassistant/helpers/event.py

```diff
@@ -383,7 +383,11 @@ def _async_track_state_change_event(
     action: Callable[[Event[EventStateChangedData]], Any],
     job_type: HassJobType | None,
 ) -> CALLBACK_TYPE:
-    """async_track_state_change_event without lowercasing."""
+    """async_track_state_change_event without lowercasing.
+
+    EVENT_STATE_CHANGED is fired on each occasion the state is updated
+    and changed, opposite of EVENT_STATE_REPORTED.
+    """
```

Comment:
I think this explanation should be in the public function `async_track_state_change_event` instead.